### DISCUSSION TITLE
Is an area chart possible?

### DISCUSSION THREAD

**Discussion by alpriest** [patrykandpatrick/vico#460] [Ideas]
I'm trying to recreate a chart from my Swift app into my Android app. I used an Area chart previously to describe the potential range of data and then a dashed line to describe the other. Using a `BrushShader` without a gradient i've been able to sort of recreate it but because the `BrushShader` shades all the way to the axis I have to use a white shader for the second line which doesn't look great. Any pointers would be great please. (I also couldn't adjust the `PathEffect` for the dashed line but I can live with that).

| Swift | Android |
|--|--|
| <img width="343" alt="Screenshot 2023-11-20 at 08 29 14" src="https://github.com/patrykandpatrick/vico/assets/221241/256d9d28-e127-464e-a5a0-b5cdbf54678b"> | <img width="343" alt="Screenshot 2023-11-20 at 08 29 19" src="https://github.com/patrykandpatrick/vico/assets/221241/ed02ab13-886e-4ad4-b15c-e18efdeed770"> |

**Comment by patrickmichalik** [comment]
Hello! I’m afraid there’s no such chart type at the moment, but we’ll be looking into adding it. We’re currently reworking several APIs to make this possible. In the meantime, depending on the specifics of the desired result, it may be possible to implement this as a `Decoration`.

**Reply by alpriest** [reply]
Thanks, I think i'll probably go with three lines showing the bounds for now and revisit in the future.

**Reply by patrickmichalik** [reply]
Sounds good. If you don’t mind, I’ll reopen this and move it to the “Ideas” category, so we can keep track of this feature request.
